.gitignore

```diff
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+node_modules/
```

Comment:
We've all been there! I'm pretty certain I committed `node_modules/` at [a hackathon project of mine](https://github.com/robzwolf/sbb-hack) a few years ago...